src/components/common/CreateUserWizard/CreateUserWizardValidation.tsx

```diff
@@ -0,0 +1,100 @@
+import { string, object, array, ValidationError } from 'yup';
```

Comment:
Shouldn't this file name be in camelCase, considering it contains no component?

Reply:
@jakubcolony Yes it should. Although it does look weird having a camel-cased file beginning with the component name sandwiched between the Pascal-cased components. How's this for a convention: naming validation files with `validation[ComponentName]`, e.g. `validationCreateUserWizard.ts`? That way validation files don't start with the component's name and will likely be grouped together near the bottom of the directory.

Reply:
Since the validation will most likely not be exported outside the component, I think it'd be fine to just call it `validation.ts`. We seem to follow this pattern with other component-specific files, such as `types.ts` in many components (Wizard, Popover, FileUpload to name a few) or `utils.ts` in SingleUserPicker. Obviously that would mean the validation files will be in between component files but I think that actually helps to create a clear structure of what depends on what
I'll leave that with you